vignettes/Status.Rmd

```diff
@@ -73,7 +73,7 @@ knitr::kable(df)
 
 This function is generally advertised as a user-friendly function since it only works with a known list of sites, parameter codes or characterisitic names, and start/end dates. 
```

Comment:
I don't see where 'this function' is defined, assuming it is readWQPqw()?

Reply:
It's under the header `## readWQPqw` (on line 72). We can change it though to:

The `readWQPqw` function is generally advertised...